app/controllers/api/v2/users_controller.rb

```diff
@@ -111,6 +112,26 @@ def update
         end
       end
 
+      api :PATCH, "/users/:id/invalidate_jwt", N_("Invalidate JWT for a specific users")
```

Comment:
```suggestion
      api :PATCH, "/users/:id/invalidate_jwt", N_("Invalidate all JWTs for the user")
```
We're invalidating _all_ JWTs for a _single_ user, right?

Reply:
for API, we are doing it for multiple users

Reply:
But the user is identified by an ID that must be unique for each user. So how is it for multiple users?

Reply:
for API, we are using the scoped search approach, so the search params in the curl command would be something like `{"seach" : "id ^ (1,2,3)"}` translating to search for id in 1, 2 ,3.

Reply:
Okay. In that case I suggest:
```suggestion
      api :PATCH, "/users/:id/invalidate_jwt", N_("Invalidate JWT for specific users")
```

Reply:
If you use the search param to get users, then there should not be any `:id` in the URL.
`api :PATCH, "/users/invalidate_jwt"`